Gems/Multiplayer/Code/Source/Components/LocalPredictionPlayerInputComponent.cpp

```diff
@@ -155,9 +155,13 @@ namespace Multiplayer
             // Discard move input events, client may be speed hacking
             if (m_clientBankedTime < sv_MaxBankTimeWindowSec)
             {
+                // Client blends from previous frame to target so here we subtract blend factor to get to that state
+                const float blendFactor = AZStd::max(0.f, input.GetHostBlendFactor());
+                const float adjustedBlendFactor = std::pow(0.2f, blendFactor);
+                const AZ::TimeMs blendMs = AZ::TimeMs(static_cast<float>(static_cast<AZ::TimeMs>(cl_InputRateMs)) * adjustedBlendFactor);
```

Comment:
This is really nice!

Reply:
Just a thought, we should factor out the blend function even though it's just a single 'pow(0.2f, value)', in case we find it's not adequate or buggy or we just want to try out different blend functions. Also do you want to clamp that blend factor between 0 and 1?

Reply:
I've had that thought on factoring out the blend function. I was going to wait to grab your changes to blending and then make a pass. I'll want to do that clamp once your changes are in since it changes how we represent blend factor in general. Right now, values above 1 are valid but rare (the pow does the clamp). Your change to send the end value will actually factor a lot of the spots I have to calculate the blend which'll be nice.